res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1757,24 +1762,24 @@ PioneerDDJSX.wheelLeds = function(value, group, control) {
         duration = engine.getValue(group, "duration"),
         elapsedTime = value * duration,
         remainingTime = duration - elapsedTime,
-        refsPerSecond = PioneerDDJSX.scratchSettings.vinylSpeed / 60,
-        speed = parseInt(refsPerSecond * 0x48),
-        wheelPos = 0x00;
+        revsPerSecond = PioneerDDJSX.scratchSettings.vinylSpeed / 60,
```

Comment:
I don't think there's any need for an abbreviation. `revolutionsPerSecond` is clearer.